<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Musteri_sil</class>
 <widget class="QWidget" name="Musteri_sil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Müşteri Silme Sayfası</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Silmek istediğiniz müşterinin TC kimlik numarasını giriniz:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sil">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_vazgec">
      <property name="text">
       <string>Vazgeç</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_sil">
      <property name="text">
       <string>Müşteriyi Sil</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
